app/assets/javascripts/discourse/app/components/discovery-topics-list.js

```diff
@@ -1,26 +1,31 @@
 import Component from "@ember/component";
+import { action } from "@ember/object";
 import { service } from "@ember/service";
 import $ from "jquery";
+import { applyBehaviorTransformer } from "discourse/lib/transformer";
 import LoadMore from "discourse/mixins/load-more";
 import { observes, on } from "discourse-common/utils/decorators";
 
 export default Component.extend(LoadMore, {
   classNames: ["contents"],
   eyelineSelector: ".topic-list-item",
+  appEvents: service(),
   documentTitle: service(),
 
   @on("didInsertElement")
   _monitorTrackingState() {
     this.stateChangeCallbackId = this.topicTrackingState.onStateChange(() =>
       this._updateTrackingTopics()
     );
+    this.appEvents.on("discovery-topics-list:loadMore", this, this.loadMore);
```

Comment:
Hmm, not really a fan of adding an appEvent here. In general, appEvents circumvent the DDAU pattern we try to follow everywhere.

Is there some other way we could make `loadMore` available to the consumer? Maybe by passing it as an argument to a plugin outlet, or as an argument to the transformer?

Reply:
Found a workaround. The event was removed from the PR.